test/libyul/yulOptimizerTests/expressionSimplifier/exp_simplifications.yul

```diff
@@ -0,0 +1,22 @@
+{
+    let t := calldataload(0)
+    sstore(0, exp(0, t))
+    sstore(1, exp(1, t))
+    sstore(2, exp(2, t))
+    // The following should not be simplified
+    sstore(3, exp(8, t))
+}
+// ====
+// EVMVersion: >=byzantium
```

Comment:
Ah, the `exp(2, X)` optimisation did not take place because bitwise shifting is present `>= constantinople` only